src/transformers/models/emformer/configuration_emformer.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2021 The Fairseq Authors and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Emformer model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+EMFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "anton-l/emformer-base-librispeech": (
```

Comment:
Can we put it maybe 
```suggestion
    "anton-l/emformer-base-librispeech": (
```

Let's move it to Facebook no?

Reply:
@anton-l can you move it to the facebook org?